data/map planets.txt

```diff
@@ -2651,6 +2708,16 @@ planet Memory
 		threshold 4000
 		fleet "Small Deep Security" 16
 
+planet "Mendez Station"
+	attributes ccor station
+	landscape land/station47
+	music ambient/machinery
+	description `Mendez Station is the closest object to Alpherg's twin stars. Constructed by the CCOR as part of its efforts to build self-sufficiency in the Outer Rim, the station's gigantic ramscoops collect charged particles from the system's solar winds and refine them into hyperdrive fuel.`
+	description `	Since its establishment, Mendez has been uninhabited for most of its operational life. The station's location makes it uncomfortably hot, and as a result, Mendez is almost entirely automated apart from a small control room used during maintenance.`
```

Comment:
Might it make sense to add a `port` to this station with no services or news and only the ability to recharge fuel?

Reply:
Gave it uninhabited. I'm not sure about giving it port for the reason you mentioned as it might be odd that Pirate ships regularly land and take off of a smaller little station, unless we lore-in that it's a fuel hub or something.

Reply:
We could create a port attribute that prevents normal NPCs from making use of them, but that's an idea for later.